### ISSUE TITLE
Notice to everyone

### ISSUE THREAD

**Issue by ybd-project-ver1** [ybd-project-ver1/ytdl-core#1]
## Details
Currently, the YBD Project account (github.com/ybd-project) that managed this ytdl-core repository has been flagged as spam, which is interfering with the operation of the repository.
We are contacting GitHub in response to this. As soon as the spam flag is lifted, we will remove this repository and resume activities in the original repository.

**Comment by kimgh06** [comment]
Thanks for your action.

**Comment by ybd-project-ver1** [comment]
We are now developing a stable ytdl-core that works in browsers, serverless, and other environments.
The changelog at the moment will be as follows Please let us know if there is anything we can do to help.

## v5.2.0 (2024/MM/DD) *Not released, planned.

### Features
* **YtdlCore:** Support for use in browsers. (To use, import `@ybd-project/ytdl-core/browser`)
* **YtdlCore:** Add a processing-optimized YtdlCore for deployment to serverless functions such as Vercel Functions (just use `@ybd-project/ytdl-core/serverless` when importing)
* **YtdlCore:** Changed to be able to import types used in YtdlCore (To use, import `@ybd-project/ytdl-core/types`)
* **YtdlCore:** Static methods such as the `getFullInfo` function have been eliminated in view of optional adaptations, etc.
* **Log:** Added `logDisplay` option to specify the type of log to display
* **Country:** Add `gl` option
* **Fetcher:** Add `fetcher` option to control requests (Allows proxy adaptation, etc. (returns a Response object))
* **Agent:** Proxy adaptation to requests by YtdlCore has been discontinued due to specification issues.

### Change
* **Lang:** Remove the `lang` option and add the `hl` option.
* **Debug:** Eliminated the use of the `YTDL_DEBUG` environment variable
* **Options:** The `requestOptions` option is obsolete.
* **Request:** Eliminate the use of the internal Undici package

**Comment by ybd-project-ver1** [comment]
A request to all of you. We are currently finishing the development and testing of v6.0.0. Therefore, I would like everyone to test v6.0.0. Please see the [v6-examples](https://github.com/ybd-project-ver1/ytdl-core/tree/main/v6-examples) folder for example implementations. As a reminder, the package.json in the implementation examples installs the latest version, so be sure to install `npm i @ybd-project/ytdl-core@beta`.

**Comment by ybd-project-ver1** [comment]
One question: I assume you receive the stream by executing the DOWNLOAD function, etc. At this time, should it return metadata such as the file extension (mp4 for video, mp3 for audio) along with the stream or just the stream?

**Comment by ybd-project-ver1** [comment]
We are releasing v6 today! Thank you for your cooperation.

**Comment by flazouh** [comment]
awesome work, thanks

**Comment by td236** [comment]
will the previous script still be maintained?
it threw an error
![image](https://github.com/user-attachments/assets/ee467215-05f6-4bbe-90c8-855e887843a3)

**Comment by td236** [comment]
will the previous issue still be updated
it threw an error
![image](https://github.com/user-attachments/assets/d0134031-ea4d-4193-b72a-23e942011ffe)

**Comment by ybd-project-ver1** [comment]
> will the previous project still be updated? it threw an error `Error: This video is unavailable`

By previous version, do you mean v5?

**Comment by td236** [comment]
i mean the YBD Project account

**Comment by ybd-project-ver1** [comment]
> i mean the YBD Project account

Yes, the old version (v5) of ytdl-core that was developed under the YBD Project account is also subject to maintenance.

**Comment by td236** [comment]
```
[ WARNING ]: Could not parse n transform function.
Please report this issue with the "1729750965169-base.js" file on https://github.com/distubejs/ytdl-core/issues.
Error: This video is unavailable
```
i  updated the latest v5 code, but cannot get the video

**Comment by ybd-project-ver1** [comment]
> ```
> [ WARNING ]: Could not parse n transform function.
> Please report this issue with the "1729750965169-base.js" file on https://github.com/distubejs/ytdl-core/issues.
> Error: This video is unavailable
> ```
> 
> i updated the latest v5 code, but cannot get the video

What is the video URL you are trying to use? Please note that we have created a new issue on this issue and you can resume the discussion there.
https://github.com/ybd-project-ver1/ytdl-core/issues/32
